keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -211,11 +218,11 @@ of selectors within a single CNP policy instance and each such selector can be c
 with its own set of peer endpoints and policy actions. This enables greater flexibility
 of configuration, and improves ease of use by not requiring cluster admins to have to come up
 with unique priority values for each instnce of rule while retaining explicit rule ordering
-as described earlier since multiple rules wihin a CNP instance are prioritized by their listing 
-order within the CNP manifest. A fully specified new rule (including selector, peer and action 
-type definitions) can hence be unambiguously inserted into an existing configuration of CNPs and 
-rules without requiring any reshuffling of priority values and while retaining unambiguous 
-relative priority ordering of all configured rules and CNP instances. 
+as described earlier since multiple rules wihin a CNP instance are prioritized by their listing
+order within the CNP manifest. A fully specified new rule (including selector, peer and action
+type definitions) can hence be unambiguously inserted into an existing configuration of CNPs and
+rules without requiring any reshuffling of priority values and while retaining unambiguous
+relative priority ordering of all configured rules and CNP instances.
 
 #### Default rules
```

Comment:
I think we can remove this section... Its somewhat obvious now that rules with a priority of "Baseline" will be be evaluated after (at a lower priority) than existing network policies